client/src/containers/CampaignPage.js

```diff
@@ -57,6 +57,12 @@ CampaignPage.propTypes = {
   fetchSignatures: PropTypes.func.isRequired,
   params: PropTypes.shape({
     id: PropTypes.string
+  }).isRequired,
+  signature: PropTypes.shape({
+    loaded: PropTypes.bool.isRequired,
+    loading: PropTypes.bool.isRequired,
+    signatures: PropTypes.arrayOf(PropTypes.object),
+    error: PropTypes.objectOf(PropTypes.any)
   }).isRequired
```

Comment:
Great example of why proptypes are good documentation, especially on an app with new folks coming and going. I'm glad we're keeping proptypes for this reason.